src/vscode-ps-rule/client.ts

```diff
@@ -88,9 +96,16 @@ export class PSRuleClient implements vscode.Disposable {
                     vscode.workspace.createFileSystemWatcher('**/'), // folder changes
                     vscode.workspace.createFileSystemWatcher('**/*.Rule.yaml'), // Rule file changes
                     vscode.workspace.createFileSystemWatcher('**/ps-rule.lock.json'), // Lock file changes
+                    vscode.workspace.createFileSystemWatcher('**/ps-rule.yaml'), // PSRule options file changes
+                    vscode.workspace.createFileSystemWatcher('**/ps-rule.yml'), // PSRule options file changes (yml)
+                    vscode.workspace.createFileSystemWatcher('**/psrule.yaml'), // PSRule options file changes (alternative name)
+                    vscode.workspace.createFileSystemWatcher('**/psrule.yml'), // PSRule options file changes (alternative name, yml)
```

Comment:
If option `PSRule.options.path` is set, watch that file, otherwise default to watching `ps-rule.yaml`.

Don't include the other variations e.g. `ps-rule.yml`, `psrule.yaml` or `ps-rule.yml`.